proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java

```diff
@@ -147,7 +150,11 @@ public void handle(Chat chat) throws Exception
         for ( int index = 0, length = chat.getMessage().length(); index < length; index++ )
         {
             char c = chat.getMessage().charAt( index );
-            Preconditions.checkArgument( c != '\u00A7' && c >= ' ' && c != 127, "illegal characters in chat" ); // Section symbol, control sequences, and delete
+            if ( c == '\u00A7' || c < ' ' || c == 127 ) // Section symbol, control sequences, and delete
+            {
+                con.disconnect( "illegal characters in chat" );
```

Comment:
maybe a translation would be better here?

Reply:
i don't think its needed. A vanilla client is not be able to send these chars

Reply:
and its the vanilla minecraft message for this case